### PR TITLE
Documentation for data tab and filter panel

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for the issue #752 
Documentation for the filter side panel and for the data tab.

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 26 sec

File: docs/src/user/manual/filters.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Filter Side Panel Documentation
+
+## Description
+The filter side panel allows users to refine and narrow down data based on specific criteria. There are different type of filters are available for customization.
+The filter side panel enhances the user experience by allowing them to focus on specific subsets of data based on their preferences, making it easier to analyze and work with the dataset effectively.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:5]: South Africa uses British spelling, please change analyze to analyse.
```suggestion
The filter side panel enhances the user experience by allowing them to focus on specific subsets of data based on their preferences, making it easier to analyse and work with the dataset effectively.
```

File: docs/src/user/manual/filters.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+## Filter
+![Filter](./img/Explore_filter.png)
+
+1. **Explore**:Open the explore page by clicking on the `Explore` on navbar.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:9]:
```suggestion
1. **Explore**: Open the explore page by clicking on `Explore` on the navigation bar.
```

File: docs/src/user/manual/filters.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+2. **Filter**: Open the filter side panel by clicking on the `Filter` option. Inside the side panel, you will find the all the filters. For each filter, interact with the available options or input fields to customize your filter criteria. Just click on the dropdown and select the filter.
+
+
+## Filter side panel
+![Filter side panel](./img/filters.png)
+
+## Filter Types
+
+### 1. Organisation Filter
+- **Filter Description:** This filter allows users to filter data according to the organization.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:19]: British spelling
```suggestion
- **Filter Description:** This filter allows users to filter data according to the organisation.
```

File: docs/src/user/manual/filters.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+- **Usage:** Select an organization from the dropdown menu to filter data specific to that organization.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:20]:
```suggestion
- **Usage:** Select an organisation from the dropdown menu to filter data specific to that organisation.
```

File: docs/src/user/manual/filters.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+    ## Organisation filter
+    ![Organisation filter](./img/Organisation_filter.png)
+
+    1. **Selecting All Organisations**:To view data from all organizations, simply check the "Select All" option within the Organisation Filter.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:25]:
```suggestion
    1. **Selecting All Organisations**: To view data from all organisations, simply check the "Select All" option within the Organisation Filter.
```

File: docs/src/user/manual/filters.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+    2. **Selecting Individual Organisations**:To filter data for specific organizations, follow these steps:
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:27]:
```suggestion
    2. **Selecting Individual Organisations**: To filter data for specific organisations, follow these steps:
```

File: docs/src/user/manual/filters.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+    Uncheck the "Select All" option if it is selected.
+    A list of available organizations will be displayed.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:29]:
```suggestion
    A list of available organisations will be displayed.
```

File: docs/src/user/manual/filters.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+    Check the checkboxes next to the organizations you want to include in the filter.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:30]:
```suggestion
    Check the checkboxes next to the organisations you want to include in the filter.
```

File: docs/src/user/manual/filters.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+
+### 2. Report Type Filter
+- **Filter Description:** This filter enables users to filter data based on report types.
+- **Usage:** Choose a report type from the list to view data associated with that report type.
+
+    ## Report filter
+    ![Report filter](./img/Report_filter.png)
+
+    1. **Report type**:The Report Type Filter is a single-select filter that allows users to filter data based on specific report types. With this filter, you can easily narrow down your dataset to view data associated with a particular report type.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:40]:
```suggestion
    1. **Report type**: The Report Type Filter is a single-select filter that allows users to filter data based on specific report types. With this filter, you can easily narrow down your dataset to view data associated with a particular report type.
```

File: docs/src/user/manual/filters.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+
+
+### 3. Property Filter
+- **Filter Description:** The property filter helps users filter data based on properties.
+- **Usage:** Select a property from the available options to view data related to that property.
+
+    ## Property filter
+    ![Property filter](./img/Property_filter.png)
+
+    1. **Select all**:Select all selects all the properties.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:50]:
```suggestion
    1. **Select all**: Select all selects all the properties.
```

File: docs/src/user/manual/filters.md [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+
+    2. **Show selected property count**:You will see the count of the property selected.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:52]:
```suggestion
    2. **Show selected property count**: You will see the count of the property selected.
```

File: docs/src/user/manual/filters.md [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+    ![Species filter](./img/Species_filter.png)
+
+    1. **Searching for Species**: Click on the Species Filter dropdown. You will see a list of available species. Start typing the name of the species you are looking for in the search box provided. As you type, the list will dynamically filter to display matching species. Select the species you want to filter by from the list.
+
+    2. **Single-Select Functionality**: The Species Filter allows you to select only one species at a time. If you select a different species, the filter will replace the previous selection with the newly chosen species.
+
+
+### 5. Year Range Filter
+- **Filter Description:** This filter allows users to narrow down data based on a specified year range.
+- **Usage:** Set a range of years to view data within that timeframe. For example, selecting a range of "2010-2020" will display data recorded between 2010 and 2020.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:69]:
```suggestion
- **Usage:** Set a range of years to view data within that time frame. For example, selecting a range of "2010-2020" will display data recorded between 2010 and 2020.
```

File: docs/src/user/manual/filters.md [hunk 13]
```diff
@@ -0,0 +1,91 @@
...
+
+    ## Year filter
+    ![Year filter](./img/Year_filter.png)
+
+    1. **Year range bar**: This is the bar which is used for settign the year range.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:74]:
```suggestion
    1. **Year range bar**: This is the bar which is used for setting the year range.
```

File: docs/src/user/manual/filters.md [hunk 14]
```diff
@@ -0,0 +1,91 @@
...
+
+    2. **From**: Input field for the from (starting)year value.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:76]:
```suggestion
    2. **From**: Input field for the from (starting) year value.
```

File: docs/src/user/manual/filters.md [hunk 15]
```diff
@@ -0,0 +1,91 @@
...
+
+    3. **To**: Input field for the to (ending)year value.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:78]:
```suggestion
    3. **To**: Input field for the to (ending) year value.
```

File: docs/src/user/manual/filters.md [hunk 16]
```diff
@@ -0,0 +1,91 @@
...
+
+### 6. Spatial Filter
+- **Filter Description:** This filter allows users to filter data based on spatial information.
+- **Usage:** Specify spatial criteria to view data located within a specific geographical area.
+
+    ## Spatial filter
+    ![Spatial filter](./img/Spatial_filter.png)
+
+    1. **Activity dropdown**: Using this you can filter the data according to the activity types.
+    2. **Biome type dropdown**:Using this you can filter the data according to the biome types.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:88]:
```suggestion
    2. **Biome type dropdown**: Using this you can filter the data according to the biome types.
```

File: docs/src/user/manual/filters.md [hunk 17]
```diff
@@ -0,0 +1,91 @@
...
+    3. **Critical biodiversity type dropdown**:Using this you can filter the data according to the critical biodiversity areas.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:89]:
```suggestion
    3. **Critical biodiversity type dropdown**: Using this you can filter the data according to the critical biodiversity areas.
```

File: docs/src/user/manual/filters.md [hunk 18]
```diff
@@ -0,0 +1,91 @@
...
+    4. **Protected area dropdown**:Using this you can filter the data according to the protected area types.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:90]:
```suggestion
    4. **Protected area dropdown**: Using this you can filter the data according to the protected area types.
```

File: docs/src/user/manual/filters.md [hunk 19]
```diff
@@ -0,0 +1,91 @@
...
+    5. **Population category dropdown**:Using this you can filter the data according to the population category types.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:91]:
```suggestion
    5. **Population category dropdown**: Using this you can filter the data according to the population category types.
```

File: docs/src/user/manual/reports.md [hunk 20]
```diff
@@ -0,0 +1,40 @@
...
+The main area of the page displays your dataset in a tabular format. Each row represents a record, and each column represents a data attribute.
+
+## Data table
+![Data table](./img/reports.png)
+
+
+## Features and Functionality
+
+
+1. **Explore**:Open the explore page by clicking on the `Explore` on navbar.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/reports.md:16]:
```suggestion
1. **Explore**: Open the explore page by clicking on `Explore` on the navigation bar.
```

File: docs/src/user/manual/reports.md [hunk 21]
```diff
@@ -0,0 +1,40 @@
+# Data Table Page Documentation
+
+## Description
+The Data Table Page is the central hub for viewing and interacting with your data. This page offers a user-friendly interface for exploring and manipulating your dataset. You can use a variety of features to customize your data view, including filters, column search, and data export options.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/reports.md:4]:
```suggestion
The Data Table Page is the central hub for viewing and interacting with your data. This page offers a user-friendly interface for exploring and manipulating your dataset. You can use a variety of features to customise your data view, including filters, column search, and data export options.
```

File: docs/src/user/manual/reports.md [hunk 22]
```diff
@@ -0,0 +1,40 @@
...
+
+
+4. **Column Search**: You can search for specific values within columns by selecting the column. This feature is especially useful when you need to locate specific data points within a large dataset. The table will dynamically filter to display selected column.
+
+5. **XLSX Format:** Click the "DOWNLOAD DATA REPORT" button to download the current dataset in XLSX format.
+6. **CSV Format:** Click the "DOWNLOAD DATA CSV" button to download the current dataset in CSV format.
+
+
+## Summary
+The Data Table Page is a comprehensive platform for exploring, analyzing, and interacting with your data. With filters, column search, and data export options, you have the tools necessary to customize your data view, find specific information, and export data in the format that suits your needs.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/reports.md:40]:
```suggestion
The Data Table Page is a comprehensive platform for exploring, analysing, and interacting with your data. With filters, column search, and data export options, you have the tools necessary to customise your data view, find specific information, and export data in the format that suits your needs.
```

File: docs/src/user/manual/reports.md [hunk 23]
```diff
@@ -0,0 +1,40 @@
...
+## Data table
+![Data table](./img/reports.png)
+
+
+## Features and Functionality
+
+
+1. **Explore**:Open the explore page by clicking on the `Explore` on navbar.
+2. **Data**: Click on the `data` button to view the data.
+3. **Filters**: Utilize the filters provided on the page to refine your data view. These filters include:
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/reports.md:18]:
```suggestion
3. **Filters**: Utilise the filters provided on the page to refine your data view. These filters include:
```

File: docs/src/user/manual/filters.md [hunk 24]
```diff
@@ -0,0 +1,91 @@
+# Filter Side Panel Documentation
+
+## Description
+The filter side panel allows users to refine and narrow down data based on specific criteria. There are different type of filters are available for customization.
+The filter side panel enhances the user experience by allowing them to focus on specific subsets of data based on their preferences, making it easier to analyze and work with the dataset effectively.
+## Filter
+![Filter](./img/Explore_filter.png)
+
+1. **Explore**:Open the explore page by clicking on the `Explore` on navbar.
+2. **Filter**: Open the filter side panel by clicking on the `Filter` option. Inside the side panel, you will find the all the filters. For each filter, interact with the available options or input fields to customize your filter criteria. Just click on the dropdown and select the filter.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:10]:
```suggestion
2. **Filter**: Open the filter side panel by clicking on the `Filter` option. Inside the side panel, you will find the all the filters. For each filter, interact with the available options or input fields to customise your filter criteria. Just click on the dropdown and select the filter.
```

File: docs/src/user/manual/reports.md [hunk 25]
```diff
@@ -0,0 +1,40 @@
...
+    - **Organisation Filter:** Filter data by organization.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/reports.md:19]:
```suggestion
    - **Organisation Filter:** Filter data by organisation.
```

File: docs/src/user/manual/filters.md [hunk 26]
```diff
@@ -0,0 +1,91 @@
+# Filter Side Panel Documentation
+
+## Description
+The filter side panel allows users to refine and narrow down data based on specific criteria. There are different type of filters are available for customization.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/filters.md:4]:
```suggestion
The filter side panel allows users to refine and narrow down data based on specific criteria. There are different type of filters are available for customisation.
```

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 5 min 52 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 33 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 35 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 5 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 5 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 56 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 20 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 16 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 12 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 7 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 26 sec

**Review by Jeremy-Prior — APPROVED** [review]
Docs LGTM

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 17 min

**Pull request merged**